styles/style.css

```diff
@@ -210,6 +210,7 @@ div#content_block { width: 70% }
 
 /*Facebook Button Frontend Styles*/
 .rtsocial-fb-button{ background-position: 0 -172px; display: block; height: 24px; width: 50px; text-indent: -9999px; text-transform: uppercase; margin: 0 auto;}
+.rtsocial-fb-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -172px no-repeat !important;}
```

Comment:
Use of `!important` is causing the PHPCS test failure.

Reply:
The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files